### PR TITLE
[REF-6034] AccuracyScore: Adding MLOps Classification Metrics - Accuracy Score

### PR DESCRIPTION
**Pull request by harshilpm**
[REF-6034] AccuracyScore: Adding MLOps Classification Metrics - Accuracy Score
Usages:
    labels_pred = [1, 0, 1, 1, 1, 0]
    labels_actual = [0, 1, 0, 0, 0, 1]

    accuracy_score = sklearn.metrics.accuracy_score(labels_actual, labels_pred)

    # first way
    pm.set_stat(ClassificationMetrics.ACCURACY_SCORE, accuracy_score)

    # second way
    pm.metrics.accuracy_score(y_true=labels_actual, y_pred=labels_pred)

    sample_weight = [0.9, 0.1, 0.5, 0.9, 1.0, 0]

    # testing with sample weights as well
    pm.metrics.accuracy_score(y_true=labels_actual,
                              y_pred=labels_pred,
                              sample_weight=sample_weight)
Testing Done:
- Unit Test

### CODE REVIEW

File: mlops/parallelm/mlops/ml_metrics_stat/accuracy_score.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        if isinstance(accuracy_score, (int, float)):
+            single_value = \
+                SingleValue() \
+                    .name(ClassificationMetrics.ACCURACY_SCORE.value) \
+                    .value(accuracy_score) \
+                    .mode(StatCategory.TIME_SERIES)
+        else:
+            raise MLOpsStatisticsException \
+                ("For outputting accuracy score, accuracy should be of type numeric but got {}."
+                 .format(type(accuracy_score)))
```
Comment (lior-amar-pm) [inline mlops/parallelm/mlops/ml_metrics_stat/accuracy_score.py:31]: Since this error is all over: ask Drew if this is the right english syntax.

**Review by lior-amar-pm — APPROVED** [review]

File: mlops/parallelm/mlops/ml_metrics_stat/accuracy_score.py (hunk 1 above)
Comment (yakov-g) [reply mlops/parallelm/mlops/ml_metrics_stat/accuracy_score.py:31]: +1

**Review by yakov-g — APPROVED** [review]

File: mlops/parallelm/mlops/ml_metrics_stat/accuracy_score.py (hunk 1 above)
Comment (harshilpm) [reply mlops/parallelm/mlops/ml_metrics_stat/accuracy_score.py:31]: i will ask her on pr : https://github.com/mlpiper/mlpiper/pull/93
